diff --git a/products/workbench/workbench_session/meta/workbench_session.cml b/products/workbench/workbench_session/meta/workbench_session.cml
--- a/products/workbench/workbench_session/meta/workbench_session.cml
+++ b/products/workbench/workbench_session/meta/workbench_session.cml
@@ -267,6 +267,7 @@
                 "fuchsia.tracing.provider.Registry",
                 "fuchsia.ui.composition.Allocator",
                 "fuchsia.ui.composition.Flatland",
+                "fuchsia.ui.composition.FlatlandFactory",
                 "fuchsia.ui.input.ImeService",
                 "fuchsia.ui.input3.Keyboard",
             ],
